installing/installing_openstack/installing-openstack-user-sr-iov.adoc

```diff
@@ -0,0 +1,92 @@
+[id="installing-openstack-user-sr-iov"]
+= Installing a cluster on OpenStack on your own SR-IOV infrastructure
+include::modules/common-attributes.adoc[]
+:context: installing-openstack-user-sr-iov
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+{rh-openstack-first} that runs on user-provisioned infrastructure and uses single-root input/output virtualization (SR-IOV) networks to run compute machines.
+
+Using your own infrastructure allows you to integrate your cluster with existing infrastructure and modifications. The process requires more labor on your part than installer-provisioned installations, because you must create all {rh-openstack} resources, like Nova servers, Neutron ports, and security groups. However, Red Hat provides Ansible playbooks to help you in the deployment process.
```

Comment:
```suggestion
Using your own infrastructure allows you to integrate your cluster with existing infrastructure and modifications. The process requires more labor on your part than installer-provisioned installations, because you must create all {rh-openstack} resources, such as Nova servers, Neutron ports, and security groups. However, Red Hat provides Ansible playbooks to help you in the deployment process.
```